### ISSUE TITLE
Legacy code stopped working

### ISSUE THREAD

**Issue by vonjd** [bips-hb/neuralnet#25]
The first example in http://blog.ephorie.de/understanding-the-magic-of-neural-networks stopped working for no apparent reason.

It is a super simple net and gave the right results with `compute`. Now neither `compute` nor `predict` work, only giving `NA`.

Please advise... Thank you

**Comment by mnwright** [comment]
In that example some weights are NA. In the old version these were just set to 0 and we are now doing the same. The example should work again.

**Comment by vonjd** [comment]
I used the current version from CRAN, which does not work... so do you mean that it is working in the github version which will be rolled out to CRAN next?

**Comment by mnwright** [comment]
Yes, you'll have to install the Github version until it is on CRAN, e.g. with

`devtools::install_github("bips-hb/neuralnet")`

**Comment by vonjd** [comment]
Yep, works! Thank you... when will the updated version be on CRAN?

**Comment by mnwright** [comment]
That might take one or two months. You could also try to avoid the NA weights to make it work with the current CRAN version.

**Comment by vonjd** [comment]
good point... why do the NAs arise in the first place in this super simple example?!?

**Comment by mnwright** [comment]
Because you asked for them with `exclude`.

**Comment by vonjd** [comment]
Because I wanted to exclude the bias neurons - see also here: https://stackoverflow.com/questions/40633567/how-to-exclude-bias-neurons-in-neuralnet-in-r

Is there a better way to do that?

**Comment by mnwright** [comment]
OK, that makes sense. I think we need a better way to build models without bias units. 

A simple fix to make your model run with the current CRAN version could also be to manually set the bias units to 0, e.g., 

````R
neuralnetwork$weights[[1]] <- lapply(neuralnetwork$weights[[1]], function(x) {
  x[is.na(x)] <- 0
  x
})
````
